build-system/test-server.js

```diff
@@ -111,5 +111,17 @@ app.use('/form/post', function(req, res) {
   });
 });
 
+/**
+ * Serve entry point script url
+ */
+app.get('/dist/ww.js', function(req, res, next) {
```

Comment:
I wonder if this is needed.
We have https://github.com/ampproject/amphtml/blob/master/build-system/tasks/karma.conf.js#L188
where we import our gulp webserver as a Karma server middleware.